testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication.py

```diff
@@ -136,7 +142,8 @@ def test_replication_configuration_valid_values(params_from_base_test_setup, num
     ('session', attachment.generate_2_png_10_10),
     ('basic', None)
 ])
-def test_replication_configuration_with_pull_replication(params_from_base_test_setup, authenticator_type, attachments_generator):
+def test_replication_configuration_with_pull_replication(params_from_base_test_setup, authenticator_type,
+                                                         attachments_generator):
```

Comment:
seem there is no change here. Could you fix the text editor issue? it will confuse people in a long run when we need to track file change history, if there is no change, it's better not showing on changelist with empty line or so.

Reply:
It is for better readability